README.md

```diff
@@ -1 +1,76 @@
-# Leaderboard-Project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard Project
+
+> In this project I have set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules. I have developed a first working version of the app following a wireframe, but without styling -Focus is on functionality. I have consumed the Leaderboard API using JavaScript async and await and added some styling.
+
+
+###### Languages 
+- HTML
+- CSS
+- JAVASCRIPT
+###### Tools  
+- Github
+###### Code Editor
+- VS code
+###### Packages 
+- NPM
+- Node
+- Webpack
+
+## Live Demo (if available)
+
+[View Live Demo Here](https://rn486.github.io/Leaderboard-Project/)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
```

Comment:
- [ ] Please remove all unnecessary titles since they are empty  or add details to them (Prerequisites, run tests Deployment ....) 🚮